Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: god mode cheat

### Description (*)

This PR adds a cheat to enable/disable god mode, preventing player characters from taking damage while active. To use it, open the cheat window with either the host or a client, and click on the "Toggle God Mode" button. This will make your character invulnerable. It will also add an indication on the character's health bar in the HUD, which is also visible for other players.
At the moment, this indicator is simply an image that is shown on top of the health bar that I selected from what we already had, but we might want a custom image for this, or a different kind of indication. See the screenshots below for how it looks on the player and an ally.

![GodModeActivatedMainCharacter](https://user-images.githubusercontent.com/90722717/149406227-672447ff-2a45-4b09-b0af-d587eeab0266.JPG)
![GodModeActivatedAlly](https://user-images.githubusercontent.com/90722717/149406225-2d966371-d320-44e0-b8ea-61995cbcdc16.JPG)

### Related Pull Requests
#444
### Issue Number(s) (*)

Fixes issue(s): [MTT-1973](https://jira.unity3d.com/browse/MTT-1973)
### Manual testing scenarios

1. Start a game
2. Open the cheat window using `/` key
3. Press the "Toggle God Mode" button
3. See that your player can no longer be damaged and that an indicator appears on their health bar in the HUD
### Questions or comments

### Contribution checklist
